src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -430,6 +461,30 @@ public Visitable visit(NameExpr expr, Void p) {
     return super.visit(expr, p);
   }
 
+  /**
+   * Updates the list of used classes based on a resolved method declaration. If the input method
+   * originates from an interface, that interface will be added to the list of used classes. The
+   * determination of whether a method belongs to an interface is based on three criteria: method
+   * name, method return type, and the number of parameters.
```

Comment:
Note that this is an over-approximation, to handle generics.

(Technically, equality between methods is defined over the _types_ of the parameters, but here we want to preserve any method that _might_ be overridden, without having to solve type parameters. This will be a good example for the paper of how Specimin is over-approximating the set of things that the verifier might use.)